packages/@okta/vuepress-site/docs/concepts/third-party-cookies/index.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Cross-Site Cookie Blocking
+---
+
+# Cross-Site Cookie Blocking
+
+Cross-site cookie blocking is a privacy-protection feature that is being added to web browsers like Safari. It can disrupt certain customer-hosted flows in Okta.
```

Comment:
Mention Firefox browser, and call out that we expect this to become common across more browsers over time. The Brave browser also blocks thrid party cookies by default today.